copier.yml

```diff
@@ -127,3 +127,11 @@ include_wkhtmltopdf:
   help:
     Do you need to install wkhtmltopdf? Usually only needed if you're going to test PDF
     report generation.
+
+ci_extra_env:
+  type: yaml
+  default: {}
+  help: |
+    Any extra environment variables to inject into the CI
+
+    Example: {"KEY": "VALUE"}
```

Comment:
So this would be a github-only question.

Reply:
Changed to only ask if ci == 'GitHub' on latest commit